sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -21,18 +21,18 @@ There is one swagger for Chat management APIs.
 
 ```ps
 cd <swagger-folder>
-autorest README.md --java --v4 --use=@autorest/java@4.0.11 --use=@autorest/modelerfour@4.15.442
+autorest README.md --java --v4 --use=@autorest/java@4.0.20 --use=@autorest/modelerfour@4.15.442
 ```
 
 ## Update generated files for chat service
 To update generated files for chat service, run the following command
 
-> autorest README.md --java --v4 --use=@autorest/java@4.0.11 --use=@autorest/modelerfour@4.15.442
+> autorest README.md --java --v4 --use=@autorest/java@4.0.20 --use=@autorest/modelerfour@4.15.442
 
 ### Code generation settings
 ``` yaml
-tag: package-chat-2021-03-07
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/896d05e37dbb00712726620b8d679cc3c3be09fb/specification/communication/data-plane/Chat/readme.md
+input-file:
```

Comment:
Is it possible to still follow the existing convention? We are supposed to update the rest-api repo first. If a new commit happened already, why wasn't the readme.md update in rest-api repo?

Reply:
I don't think so, the new swagger from the rest-api specs (**https://raw.githubusercontent.com/Azure/azure-rest-api-specs/08ef4099dff05de3ce1682d58cd4e0e2b1565905/specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2021-04-05-preview6/communicationserviceschat.json**) doesn't seem compatible with that convention anymore. 

C# also made this convention change to update to the 2021-04-05-preview6 version: https://github.com/Azure/azure-sdk-for-net/pull/21097/files?file-filters%5B%5D=.cs&file-filters%5B%5D=.json&file-filters%5B%5D=.md#diff-854b8aa8c3656ca009a15c7a2bb49f67ea5ecfc537397345863a6a1b7c091e27